client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -156,9 +157,11 @@ class TaxonomyManagerListItem extends Component {
 							{ translate( 'Delete' ) }
 						</PopoverMenuItem>
 					}
-					<PopoverMenuItem href={ this.getTaxonomyLink() } icon="external">
-						{ translate( 'View Posts' ) }
-					</PopoverMenuItem>
+					{ ! isJetpack &&
+						<PopoverMenuItem href={ this.getTaxonomyLink() } icon="external">
```

Comment:
I like this idea, but should it be a term selector since it is a term oriented link?  I'll just do a follow-up PR though.  I did forget to mention that I created an API diff to add the needed values to the site response in code-D3665